frontend/src/app/modules/router/base/application-base.component.ts

```diff
@@ -27,7 +27,6 @@
 // ++
 
 import {Component} from "@angular/core";
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 
 export const appBaseSelector = 'openproject-base';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/modules/router/base/application-base.component.ts#L32"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>